pkg/metaserver/agent/metric/fake_metric.go

```diff
@@ -99,6 +99,10 @@ func (f *FakeMetricsFetcher) GetDeviceMetric(deviceName string, metricName strin
 	return f.metricStore.GetDeviceMetric(deviceName, metricName)
 }
 
+func (f *FakeMetricsFetcher) GetDeviceStatMetric(deviceName string, metricName string) (metric.MetricData, error) {
```

Comment:
Do not need this interface if we combine raw and rate metrics store together. Same as SetDeviceStatMetric.